database/database.go

```diff
@@ -243,6 +243,31 @@ func (db *DB) Purge(ctx context.Context) error {
 	return nil
 }
 
+// Hashes returns all hashes in our database that have not been marked as failed
+func (db *DB) Hashes() ([]Hash, error) {
+	// Find all documents
+	filter := bson.M{
+		"hash":   bson.M{"$ne": nil},
+		"failed": bson.M{"$ne": true},
+	}
+
+	opts := options.Find()
+	opts.SetSort(bson.D{{"timestamp_added", 1}})
```

Comment:
There used to be but in the new architecture, there's no reason for them to be sorted. For what it's worth though, if you're sorting on an indexed field (which is the case here) in mongo the overhead is really very very minimal because it can just use its index which is sorted already.

That said, I was able to remove it because we got rid of the `latestBlockTime` timestamp thing.